pkg/controller/build/build_controller.go

```diff
@@ -111,13 +114,14 @@ func (r *ReconcileBuild) Reconcile(request reconcile.Request) (reconcile.Result,
 	ctx, cancel := context.WithTimeout(r.ctx, r.config.CtxTimeOut)
 	defer cancel()
 
-	ctxlog.Info(ctx, "reconciling Build", "Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	ctxlog.Debug(ctx, "start reconciling Build", namespace, request.Namespace, name, request.Name)
```

Comment:
Why we change this type to the `Debug` type? I think this message can help us know and find there is a reconciliation start.

Do you think it is reported so many times?

Reply:
yes, this is too much for INFO.